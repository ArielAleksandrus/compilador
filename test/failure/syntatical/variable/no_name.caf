int soma(int a[]){
    int [];
    retorne 0;
}

programa{}
