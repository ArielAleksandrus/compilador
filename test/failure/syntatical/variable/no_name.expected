ERROR => Line 2, col 9: Expected name of variable, got: Operator
